src/lib.rs

```diff
@@ -129,23 +129,14 @@ This library supports four features:
 #[path = "../proto/proto_model.rs"]
 pub mod proto;
 
-#[cfg(feature = "protobuf")]
-macro_rules! from_vec {
-    ($e: expr) => {
-        ::protobuf::RepeatedField::from_vec($e)
-    };
-}
-
 #[cfg(not(feature = "protobuf"))]
 #[path = "plain_model.rs"]
 pub mod proto;
 
-#[cfg(not(feature = "protobuf"))]
-macro_rules! from_vec {
-    ($e: expr) => {
-        $e
-    };
-}
+#[cfg(feature = "protobuf")]
+mod convert;
```

Comment:
Unless you have a strong feeling about keeping these two split, I'd move everything into a single `proto_ext` module.